arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+                    <title>`9RED BED DESTROYED</title>
+                    <subtitle>`6Your team lost the round</subtitle>
+                </message>
+                <message text="`7`lYour team lost the round!"/>
+            </switch-scope>
+        </switch-scope>
+    </action>
+    <!-- show to Blue team on a round Loss -->
+    <trigger filter="blue-team-lose" action="blue-team-lose-action" scope="player"/>
+    <action id="blue-team-lose-action" scope="player">
+        <switch-scope outer="player" inner="match">
+            <switch-scope outer="match" inner="team" filter="only-team-two">
+                <set var="blue_team_lose" value="0"/>
+                <sound key="mob.enderdragon.growl" volume="1" pitch="1"/>
+                <sound key="mob.enderdragon.growl" volume="1" pitch=".9"/>
+                <sound key="mob.enderdragon.growl" volume="1" pitch=".8"/>
+                <sound key="objective_bad" volume="1" pitch="1"/>
+                <message fade-in="0.5s" stay="3s" fade-out="0.5s">
+                    <title>`9BLUE BED DESTROYED</title>
+                    <subtitle>`6Your team lost the round</subtitle>
+                </message>
+                <message text="`7`lYour team lost the round!"/>
+            </switch-scope>
+        </switch-scope>
+    </action>
+    <trigger filter="red-team-win-final" scope="player">
+        <action scope="player">
+            <set var="red_team_win" value="0"/>
+            <set var="blue_team_lose" value="2"/>
+            <set var="gameover" value="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".9"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".8"/>
+        </action>
+    </trigger>
+    <trigger filter="blue-team-win-final" scope="player">
+        <action scope="player">
+            <set var="blue_team_win" value="0"/>
+            <set var="red_team_lose" value="2"/>
+            <set var="gameover" value="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch="1"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".9"/>
+            <sound key="ambient.weather.thunder" volume="1" pitch=".8"/>
+        </action>
+    </trigger>
+    <trigger filter="red-final" scope="team">
+        <action scope="team">
+            <switch-scope outer="team" inner="match">
+                <action scope="match">
+                    <set var="red_team_win" value="2"/>
+                </action>
+            </switch-scope>
+        </action>
+    </trigger>
+    <trigger filter="blue-final" scope="team">
+        <action scope="team">
+            <switch-scope outer="team" inner="match">
+                <action scope="match">
+                    <set var="blue_team_win" value="2"/>
+                </action>
+            </switch-scope>
+        </action>
+    </trigger>
```

Comment:
```suggestion
    <trigger filter="blue-final" scope="team">
        <action>
            <set var="blue_team_win" value="2"/>
        </action>
    </trigger>
```
A bit of redundant stuff:
 - If the scope is defined in a trigger, the an inner <action> is already assumed of the same scope, so it's not needed
 - The switch-scope is not needed, since the variables of higher scopes can be accessed from lower scopes (ie: from team scope you can modify team, or match scoped variables)

Reply:
Clean, thank you